Dockerfile.prod

```diff
@@ -0,0 +1,34 @@
+FROM python:3.10
+
+ENV PYTHONUNBUFFERED 1
+ARG REQUEST_BROKER_DNS
+ARG APPLICATION_PORT
+RUN apt-get update
+RUN apt-get install --yes apache2 apache2-dev 
+RUN apt-get install --yes postgresql
+RUN apt-get -y install python3-pip
+RUN pip install --upgrade pip
+
+RUN wget https://github.com/GrahamDumpleton/mod_wsgi/archive/refs/tags/4.9.0.tar.gz \
+    && tar xvfz 4.9.0.tar.gz \
+    && cd mod_wsgi-4.9.0 \
+    && ./configure --with-apxs=/usr/bin/apxs --with-python=/usr/local/bin/python \
+    && make \
+    && make install \
+    && make clean
+
+ADD ./apache/000-request_broker.conf /etc/apache2/sites-available/000-request_broker.conf
+RUN sed "s/ENV_REQUEST_BROKER_DNS/${REQUEST_BROKER_DNS}/" -i /etc/apache2/sites-available/000-request_broker.conf
+RUN sed "s/ENV_REQUEST_BROKER_PORT/${APPLICATION_PORT}/" -i /etc/apache2/sites-available/000-request_broker.conf
+ADD ./apache/wsgi.load /etc/apache2/mods-available/wsgi.load
+RUN a2ensite 000-request_broker.conf
+RUN a2enmod headers
+RUN a2enmod rewrite
+RUN a2enmod wsgi
+
+COPY . /var/www/request-broker
+WORKDIR /var/www/request-broker
+RUN pip install --no-cache-dir -r requirements.txt
+
+EXPOSE ${APPLICATION_PORT}
+ENTRYPOINT ["/var/www/request-broker/entrypoint.sh", "apache"]
```

Comment:
The issue with using the existing entrypoint.sh is that the `wait-for-it.sh` check for database status has a hardcoded host and port. This raises the question of whether this Dockerfile is intended to be used in conjunction with a compose file. If so, it may make sense to provide a prod compose file in addition to this Dockerfile.